.appveyor.yml

```diff
@@ -8,6 +8,8 @@ install:
     (New-Object Net.WebClient).DownloadFile('https://github.com/electron/rcedit/releases/download/v1.1.1/rcedit-x86.exe', 'rcedit-x86.exe')
 
     $env:PATH="$env:PATH;C:\lazarus;$((Get-ChildItem -Recurse C:\lazarus\fpc\strip.exe).DirectoryName);C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin"
+before_build:
+- nuget restore Source\ORTS.sln
```

Comment:
Sorry, this change is not in line with how we build Open Rails - you should make all modifications within `Build.cmd` alone.